runtime/port/unix/j9sock.c

```diff
@@ -1857,8 +1857,8 @@ j9sock_getopt_sockaddr(struct J9PortLibrary *portLibrary, j9socket_t socketP, in
 
 	/* if IPv4 the OS returns in_addr, if IPv6, value of interface index is returned */
 	typedef union byte_or_int {
-		uint8_t byte;
-		uint32_t integer;
+		uint8_t byte_t;
+		uint32_t integer_t;
```

Comment:
Names ending with `_t` normally denote types. Can you change these names? Perhaps `byte_val` and `integer_val`?